### ISSUE TITLE
Empathy Map & User Persona

### ISSUE THREAD

**Issue by Enigmas01** [zuri-training/anima_lib_team-91#17]
This is an empathy map and user persona drawn from our research for our animation library project called StyleFlow.

**Comment by Enigmas01** [comment]
https://www.figma.com/file/A92MtsaU9yIJukBslt8CNY/StyleFlow-Board

**Comment by Sun-base** [comment]
All tasks have been successfully completed from the user research to the features.

**Comment by Mojezcr8tiv** [comment]
On this task, I worked on one user persona and story. I also worked on the empathy map, generating think, say, feel, do from the results of the interviews conducted by some of my team members. I also generated some ideas and possible solutions to the problems of the intended users.

**Comment by amandaoli** [comment]
I conducted an interview with a candidate, organized interview feedback from the candidate and created the aggregated empathy map (think, say, feel, do). Also, I brainstormed to come up with possible features I believe prospective users need.

**Comment by Taiwo93** [comment]
It was great working with this team 
With all hands on desk we all use what we gathered from our research to come up with an Empathy map and User persona

**Comment by Taiwo93** [comment]
https://www.figma.com/file/A92MtsaU9yIJukBslt8CNY/StyleFlow-Board?node-id=24%3A40

https://www.figma.com/file/A92MtsaU9yIJukBslt8CNY/StyleFlow-Board?node-id=24%3A48

https://www.figma.com/file/A92MtsaU9yIJukBslt8CNY/StyleFlow-Board?node-id=24%3A56

https://www.figma.com/file/A92MtsaU9yIJukBslt8CNY/StyleFlow-Board?node-id=24%3A64

https://www.figma.com/file/A92MtsaU9yIJukBslt8CNY/StyleFlow-Board?node-id=24%3A49

https://www.figma.com/file/A92MtsaU9yIJukBslt8CNY/StyleFlow-Board?node-id=24%3A57

https://www.figma.com/file/A92MtsaU9yIJukBslt8CNY/StyleFlow-Board?node-id=24%3A65

**Comment by Taiwo93** [comment]
https://www.figma.com/file/A92MtsaU9yIJukBslt8CNY/StyleFlow-Board?node-id=24%3A43

**Comment by Enigmas01** [comment]
Working with some of my colleagues on this particular task was such a bliss. I worked on multiple frames, created the template we were to work on. I analyzed the interview feedback that was conducted by my team mate to get to understand our potential user's problems, with the interview feedback, I created one user persona for a female user with hypothetical name. Created the pain point section. Put up some possible features that would appear on our project's website. Populated the Brainstorming section using the feedback I got from other members of the team after a general meeting we had on Google Meet.
